Client/Frontend/Settings/SearchSettingsTableViewController.swift

```diff
@@ -201,6 +202,9 @@ class SearchSettingsTableViewController: ThemedTableViewController, FeatureFlagg
         let section = Section(rawValue: indexPath.section) ?? .defaultEngine
         switch section {
         case .defaultEngine:
```

Comment:
Did you check if this also affects the current production version? Given it's related to the search shortcuts experiment I guess it might 🤔